docs/news/news-v5.0.x.rst

```diff
@@ -4,6 +4,42 @@ Open MPI v5.0.x series
 This file contains all the NEWS updates for the Open MPI v5.0.x
 series, in reverse chronological order.
 
+Open MPI version v5.0.1rc1
+--------------------------
+:Date: 11 December 2023
+
+- Internal PMIx and PRRTe versions:
+
+  - PMIx (v4.2.8rc1-1-g0e09e062). Commit hash: ``0e09e062e8cefe0feb18ea37d768ded213053d68``.
+  - PRRTE (v3.0.3rc1-2-g33aa50909f). Commit hash: ``33aa50909fe2c223f9f515fae120cd6a21d04653``.
+ 
+- Bugfixes and changes (in chronological order)
+    
+  - docs: update to match PRRTE v3.0.3 docs
+          update static application build docs
+          Many many other documentation related updates and changes
```

Comment:
Looking at this again, I think we only need 1 sentence, e.g. `Various documentation updates`. Otherwise looking good!

Reply:
done - fixed, forced pushed